src/backend/egl/surface.rs

```diff
@@ -143,11 +143,13 @@ impl EGLSurface {
     pub fn swap_buffers(
         &self,
         damage: Option<&mut [Rectangle<i32, Physical>]>,
+        damage_impl: DamageSupport,
```

Comment:
Is there a reason someone would want to select the damage implementation?

I'd think it would be more ergonomic to just select the damage method internally?

Reply:
Granted. Should be fixed